src/components/modebar/manage.js

```diff
@@ -117,16 +118,16 @@ function getButtonGroups(gd, buttonsToRemove, buttonsToAdd) {
     var allAxesFixed = areAllAxesFixed(fullLayout),
         dragModeGroup = [];
 
-    if((hasCartesian || hasGL2D) && !allAxesFixed) {
+    if(((hasCartesian || hasGL2D) && !allAxesFixed) || hasTernary) {
         dragModeGroup = ['zoom2d', 'pan2d'];
     }
-    if(hasCartesian && isSelectable(fullData)) {
+    if((hasCartesian || hasTernary) && isSelectable(fullData)) {
         dragModeGroup.push('select2d');
         dragModeGroup.push('lasso2d');
     }
     if(dragModeGroup.length) addGroup(dragModeGroup);
 
-    if((hasCartesian || hasGL2D) && !allAxesFixed) {
+    if((hasCartesian || hasGL2D) && !allAxesFixed && !hasTernary) {
```

Comment:
I guess existing zoom layout updates don't generalize to the ternary `aaxis`, `baxis` and `caxis`.

It would be nice to have these buttons for ternary at some point though.